src/tools/quemap/qlight.c

```diff
@@ -83,6 +85,34 @@ _Bool Light_PointPVS(const vec3_t org, byte *pvs) {
 	return true;
 }
 
+/**
+ * @brief
+ */
+_Bool Light_InPVS(const vec3_t p1, const vec3_t p2) {
+	byte pvs[MAX_BSP_LEAFS >> 3];
+
+	const int32_t leaf1 = Cm_PointLeafnum(p1, 0);
+	const int32_t leaf2 = Cm_PointLeafnum(p2, 0);
+
+	const int32_t area1 = Cm_LeafArea(leaf1);
+	const int32_t area2 = Cm_LeafArea(leaf2);
+
+	if (!Cm_AreasConnected(area1, area2)) {
+		return false;    // a door blocks sight
+	}
+
+	const int32_t cluster1 = Cm_LeafCluster(leaf1);
+	const int32_t cluster2 = Cm_LeafCluster(leaf2);
+
+	Cm_ClusterPVS(cluster1, pvs);
+
+	if ((pvs[cluster2 >> 3] & (1 << (cluster2 & 7))) == 0) {
+		return false;
+	}
+
+	return true;
```

Comment:
Does `-light` in general work on non-VIS'ed maps? If so, it would be great if this worked on non-VIS'ed maps, too. It'd be slow, but I think it should still work.